calculator-app/src/main/java/camp/nextstep/edu/calculator/MemoryAdapter.kt

```diff
@@ -0,0 +1,33 @@
+package camp.nextstep.edu.calculator
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import camp.nextstep.edu.calculator.databinding.ItemResultBinding
+import camp.nextstep.edu.calculator.domain.data.Memory
+
+class MemoryAdapter(
+    var items: List<Memory> = emptyList()
```

Comment:
```suggestion
    val items: List<Memory> = emptyList()
```

var로 받지 않는 것이 더 혼동을 줄일 수 있을 것 같네요..!

Reply:
코드를 수정하려고 하다보니 
계산식이 추가될때마다 items가 변경해야되어야 합니다.
val로 바꾸게 되면 계산식이 추가될때마다 Adapter를 따로 생성해야하는데
val로 바꿀수 있는 좋은 방법이 있을까요? ㅠㅠ

Reply:
현재는 MemoryAdapter에 있는 값을 다른 객체들에서 변경할 수 있는 상태입니다.
이런 경우 캡슐화가 잘 진행되어있지 않게 되네요..!
객체가 가지고있는 속성들은 객체 스스로만 변경할 수 있고, 다른 객체들은 접근만 할 수 있도록 설계하는 것이 객체지향적인 관점에 더 잘 어울립니다.

힌트로는..! backing properties에 관련해서 찾아보시면 좋을 것 같습니다 :)

Reply:
items를 업데이트 하기 위해 updateItems 함수를 만들고, 
items는 private으로 변경하였습니다.
리뷰어님 의도에 맞게 수정한건지 궁금합니다.